src/modules/accommodations/api/list-accommodations.api.ts

```diff
@@ -21,17 +22,11 @@ interface AccommodationListResponseData {
   metadata: AccommodationListResponseDataMetadata;
 }
 
-interface AccommodationListResponse {
-  status: number;
-  message: string;
-  data: AccommodationListResponseData;
-}
-
 async function getAccommodations(pageParam: number, limit: number, params: ListAccommodationQueryParams) {
   const stringifiedParams = Object.fromEntries(Object.entries(params).map(([key, value]) => [key, String(value)]));
   const searchParams = new URLSearchParams(stringifiedParams).toString();
 
-  const response = await apiClient.get<AccommodationListResponse>(
+  const response = await apiClient.get<BaseResponse<AccommodationListResponseData>>(
```

Comment:
This also, please rename it to ```AccommodationList```. Inside ```BaseResponse```, it's better if we specify what exactly is coming inside ```data``` field